tests/test_userhistory.py

```diff
@@ -1,33 +1,44 @@
-""" Tests for user history class """
+''' Tests for user history class '''
 import __init__
 from userhistory import UserHistory
 
 def test_userhistory_init():
-    """Tests user history init"""
-    # specifying a short date range prevents unnecessarily pulling down
-    # a very large number of revisions - saves us time on testing
-    user_history = UserHistory("Jimbo Wales",
-                               startyear=2022, startmonth=1, startday=1,
-                               endyear=2022, endmonth=1, endday=1)
+    '''Tests user history init'''
+    user_history = UserHistory("QuicoleJR")
 
-    assert user_history.user == "Jimbo Wales"
+    assert user_history.user == "QuicoleJR"
+    assert user_history.tags is None
+    assert user_history.keyword is None
+    assert len(user_history.revisions) > 0
 
 def test_userhistory_tag_filter():
-    """Tests user history filtering"""
-    user_history = UserHistory("Jimbo Wales",
-                               startyear=2021, startmonth=5,
-                               endyear=2021, endmonth=7,
-                               tags=["mw-reverted"])
-
-    assert len(user_history.revisions) == 3
+    '''Tests user history filtering'''
+    user_history = UserHistory("QuicoleJR", tags=["mobile edit"])
```

Comment:
Given current network overhead and potential upcoming changes to default behavior, consider specifying consistent start and end dates/times.
Additionally it may be worth checking the expected behavior for a request that matches no revisions - this is something I tweaked recently in my code but failed to warn y'all about, sorry! Fortunately it didn't appear to break existing tests, but this is probably more of a condemnation of the thoroughness of our test suite than it is a happy footnote.

Reply:
Updated to use consistent timestamps and also check for no revision results